### PR TITLE
AO3-4303 i18n coauthor_notification

### PR DESCRIPTION
**Pull request by zz9pzza**
https://otwarchive.atlassian.net/browse/AO3-4303

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -40,3 +40,11 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    coauthor_notification:
+      subject: "[%{app_name}] Co-Author Notification"
+      edit_the: "Edit the %{type}"
```
Comment (cesy) [inline config/locales/en.yml]: We need to make it clear to translators what type can be - how about "edit_the_work_chapter_or_series"? It's long, and there is virtue in brevity, but they'll need the clarity.

Comment (cesy) [inline config/locales/en.yml]: And a bigger complication - what about languages in which, say, work is male and chapter is female, or something, so "the" changes? They'll need multiple options here.

File: config/locales/en.yml [hunk 2]
```diff
@@ -40,3 +40,11 @@ en:
...
+      part1: "You have been listed as a coauthor on the following %{type}:"
```
Comment (priscilladc) [inline config/locales/en.yml]: For some languages, the word for "following" will also change depending on the %{type}, just like "the" (as Cesy mentioned above, and also in line 50). I believe we'll need full strings for each different type.

File: app/mailers/user_mailer.rb [hunk 3]
```diff
@@ -264,7 +264,7 @@ def coauthor_notification(user_id, creation_id, creation_class_name)
     I18n.with_locale(Locale.find(@user.preference.preferred_locale).iso) do
       mail(
         to: @user.email,
-        subject: "[#{ArchiveConfig.APP_SHORT_NAME}] Co-Author Notification"
+        subject: "#{t 'user_mailer.coauthor_notification.subject', app_name: ArchiveConfig.APP_SHORT_NAME}"
```
Comment (sarken) [inline app/mailers/user_mailer.rb]: This should probably be just `t('user_mailer.coauthor_notification.subject', app_name: ArchiveConfig.APP_SHORT_NAME)` like the others

File: app/views/user_mailer/coauthor_notification.html.erb [hunk 4]
```diff
@@ -1,15 +1,11 @@
 <% content_for :message do %>
   <% creation_type = @creation.class.name.underscore.humanize.downcase %>
 
-  <p>
-    You have been listed as a coauthor on the following <%= creation_type %>:
-  </p>
+  <p><%= t ".part1_#{creation_type}" %></p>
```
Comment (sarken) [inline app/views/user_mailer/coauthor_notification.html.erb]: Ooh, very nice -- I like this solution!

File: config/locales/en.yml [hunk 5]
```diff
@@ -54,16 +54,26 @@ en:
...
           Cheers, 
           %{archive_name}
+    coauthor_notification:
+      subject: "[%{app_name}] Co-Author Notification"
+      edit_the_work: "Edit the work"
+      edit_the_series: "Edit the series"
+      part1_work: "You have been listed as a coauthor on the following work:"
+      part1_series: "You have been listed as a coauthor on the following series:"
+      html:
+        part2_work: "%{edit} to remove yourself as author if you've been added in error or don't want to be listed as a creator."
```
Comment (sarken) [inline config/locales/en.yml]: As much as it pains me, the "creator" at the end of all these should stay "author" unless we're going to update the whole thing to use "creator" and "co-creator". (And if we do, pretty please let's make the capitalization in the subject "Co-creator notification" -- we use a lot of sentence case subject lines and they look _so_ much better.)

File: app/views/user_mailer/coauthor_notification.html.erb (hunk 4 above)
Comment (zz9pzza) [inline app/views/user_mailer/coauthor_notification.html.erb]: I thought it was clever :)

**Comment by sarken** [comment]
Sooooooo, this is fun. It looks like merging an i18n pull request will make all the other i18n pull requests grey. /o\ This and #2117 are marked for 0.9.177, so they're the ones to focus on keeping mergeable.

**Comment by sarken** [comment]
I just merged the co-author email, so this is grey again.

**Comment by sarken** [comment]
Sorry, I mean I merged the "email changed" email. *This* is the coauthor email. :coffee:

**Pull request merged**